lib/Controller/AppSettingsController.php

```diff
@@ -120,6 +137,24 @@ public function setSpreedSettings($stun_server, $turn_server, $turn_server_secre
 			$this->config->setAppValue('spreed', 'turn_server_protocols', $turn_server_protocols);
 		}
 
+		$currentSignalingServer = $this->config->getAppValue('spreed', 'signaling_server', '');
+		if ( $currentSignalingServer !== $signaling_server ) {
+			$this->config->setAppValue('spreed', 'signaling_server', $signaling_server);
+		}
+
+		$currentSignalingSkipVerifyCert = $this->config->getAppValue('spreed', 'signaling_skip_verify_cert', '');
+		if ($signaling_skip_verify_cert === null) {
```

Comment:
```
$signaling_skip_verify_cert = (string) $signaling_skip_verify_cert;
```

Reply:
Sorry, I don't understand what you want to have changed here.

Reply:
You set the variable to an empty string when it is null, so you can also just cast it to string, getting rid of the if-block. Should have the same result?

Reply:
php magic :see_no_evil: